modules/hitlnext/src/backend/repository.ts

```diff
@@ -474,7 +474,7 @@ export default class Repository {
   }
 
   deleteHandoff = async (id: string) => {
-    return this.bp.database.transaction(async trx => {
+    this.bp.database.transaction(async trx => {
```

Comment:
Actually, the return should stay there, if you await deleteHandoff you expect it to be completed afterwards. Also, can you fix typings ? Should return a void promise